static/migrate.html

```diff
@@ -0,0 +1,103 @@
+
+    <link rel="icon" href="favicon-16.png" sizes="16x16" />
+    <link rel="icon" href="favicon-32.png" sizes="32x32" />
+    <link rel="icon" href="favicon-48.png" sizes="48x48" />
+    <link rel="shortcut icon" href="favicon.ico" />
+
+    <script src="vendor/angular/angular.js"></script>
+    <script src="vendor/angular-sanitize/angular-sanitize.js"></script>
+    <script src="vendor/jquery/dist/jquery.min.js"></script>
+    <script src="vendor/bootstrap/dist/js/bootstrap.min.js"></script>
+    <script src="vendor/angular-bootstrap/ui-bootstrap-tpls.js"></script>
+    <script src="migrate.js"></script>
+  </head>
+  <body ng-controller="MigrateControl" ng-class="loading ? 'loading' : ''">
+    <div class="titlebar">
+      <a href="https://www.sparkpost.com">
+        <img class="navbar-logo" src="https://developers.sparkpost.com/images/logo-sparkpost-white.png" alt="SparkPost" width="150">
+      </a>
+    </div>
+    <div class="container-fluid">
+      <div class="jumbotron">
+        <div class="row">
+          <div class="col-md-8 col-md-offset-2">
+              <p>From here you can migrate a template from SendGrid to SparkPost.<br/>You will need a few things to hand to get started:</p>
+              <table class="table-migrate-notes">
+                <tr>
+                  <th class="pull-right">SendGrid API Key</th>
+                  <td>Requires <strong>Template Engine</strong> and <strong>Marketing Campaigns</strong> permissions</td>
+                </tr>
+                <tr>
+                  <th class="pull-right">SendGrid Template/Marketing Campaign ID</th>
+                  <td>If importing a Marketing Campaign, provide Campaign ID; otherwise Template ID</td>
+                </tr>
+                <tr>
+                  <th class="pull-right">SparkPost API Key</th>
+                  <td>
+                    Requires <code>Templates Read/Write</code> privileges<br>
+                    <em>Note: If you're running the SparkPost Heroku addon, we have your API key in hand already.</em>
+                  </td>
+                </tr>
+
+              </table>
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-md-12">
+            <uib-alert ng-repeat="alert in alerts" type="{{alert.type}}" close="closeAlert($index)">
+              <div ng-bind-html="alert.msg"></div>
+            </uib-alert>
+            <form name="migrateFrm" ng-submit="migrate(migrateFrm.$valid)" novalidate>
+              <div class="form-group">
+                <label for="sgAPIKey">SendGrid API key <span class="text-warning" ng-show="migrateFrm.sgAPIKey.$touched && migrateFrm.sgAPIKey.$invalid">8+ chars required</span></label>
+                <input ng-disabled="loading" type="text" ng-model="sgAPIKey" ng-minlength="8" required class="form-control" name="sgAPIKey" />
+              </div>
+              <div class="form-group">
+                <div class="checkbox">
```

Comment:
I didn't do it. Let me know if you still think this way after this revision. I can modify it.